src/main/java/com/github/fakemongo/impl/aggregation/Sample.java

```diff
@@ -4,19 +4,33 @@
 import com.mongodb.DBCollection;
 import com.mongodb.DBObject;
 import com.mongodb.annotations.ThreadSafe;
-import java.util.ArrayList;
+
+import java.util.HashSet;
 import java.util.List;
+import java.util.ArrayList;
 import java.util.Random;
+import java.util.Set;
 
 /**
  */
 @ThreadSafe
 public class Sample extends PipelineKeyword {
   public static final Sample INSTANCE = new Sample();
 
-  private static final Random RANDOM = new Random();
+  private static final Random rnd = new Random();
```

Comment:
I'm asking myself if he need to be uppercase or not..

http://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

not sure..